06-building-a-program-start-to-finish/writing-tests/output/inverted-index-test.js

```diff
@@ -0,0 +1,143 @@
+const invertedObject = new InvertedIndex();
+
+
+const correctBook = require('./book.json');
+const wrongBook = require('./wrongFormat.json');
+const zeroLength = require('./length.json');
```

Comment:
'wrongBook', 'rightFile' and 'zeroLength' are assigned values but never used. (no-unused-vars)